### DISCUSSION TITLE
Projet cubli

### DISCUSSION THREAD

**Discussion by nonorobolive** [community/community#77341] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Bonjour à tous,
Je regarde de prêt cet excellent projet du cube qui tient seul en équilibre. Etant novice en programmation Arduino, je ne comprend pas comment entrer les codes fournis dans le projet. Il y a un code arduino_cube.h , un arduino_cube.ino et un functions.ino.
Je ne sais pas comment faire pour entrer le code dans une Arduino Nano ?
Si quelqu'un peut m'aider.
Merci d'avance

**Comment by queenofcorgis** [comment]
Hi @nonorobolive , thanks for participating!
We take our [Code of Conduct](https://github.com/community/community/blob/0ca37cf594e438fd4aec3c7f3736b95e6b0b3567/CODE_OF_CONDUCT.md) very seriously and want to help ensure that everyone has a good experience free of antagonism and harassment. Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be locking any posts in languages other than English for now, including this one.
